printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md

```diff
@@ -0,0 +1,43 @@
+FYSETC Spider DIN Rail Mount
+
+for horizontal or vertical mounting
+
+![FYSETC_Spider_DIN_Mount](./IMG/main.jpg)
+
+Required material:
+
+2x printed DIN mounts
+
+for horizontal mounting:
+spider_bracket_horizontal_terminal.stl
+spider_bracket_horizontal_usb.stl
+
+for vertical mountin:
```

Comment:
Replace `mountin` with `mounting`.  Either add `<br/>` at the end of these lines or use the list syntax as above.